lib/ecmascript.ts

```diff
@@ -1713,21 +2028,23 @@ export function ToTemporalCalendar(calendarLike) {
   return new TemporalCalendar(calendar);
 }
 
-function GetTemporalCalendarWithISODefault(item) {
+function GetTemporalCalendarWithISODefault(
+  item: Temporal.CalendarProtocol | { calendar?: Temporal.PlainDateLike['calendar'] } | undefined
```

Comment:
Why isn't the second thing here just `Temporal.Calendar` ? Can this also be a string identifier?

Reply:
This can be a property bag (one of the Temporal.XxxLike types) or (maybe, not sure) a Temporal object that has a calendar. This `{ calendar?: Temporal.PlainDateLike['calendar'] }` type covers all those cases. Look at references to this function and should be clearer how it's used. LMK if still unclear.

Reply:
Ah gotcha.